lms/services/aes.py

```diff
@@ -38,6 +38,9 @@ def build_iv():
         # Using the cryptographic secure Cryptodome.Random instead of the stlib random package.
         return Random.new().read(AES.block_size)
 
+    class AESSecret:
+        """Dummy type to identify strings that must be encrypted."""
```

Comment:
This is really far from where it's used. I'm not sure this makes sense on the service. 

I think this might be better defined where it's used.